sympy/ntheory/digits.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function, division
+
+import math
+import re
+
+from collections import Counter
+
+
+def count_digits(n, base=10):
+    """
+    A frequency counter for the digits of an integer ``n`` given in a base
+    ``base``.
+    """
```

Comment:
IMO, examples should be added.

Reply:
Yes, examples will added in the next commit.